indico/modules/events/registration/models/forms.py

```diff
@@ -191,6 +191,12 @@ class RegistrationForm(db.Model):
         nullable=False,
         default=''
     )
+    #: If the completed registration email should include the event's iCalendar file.
+    complete_registration_attach_ical = db.Column(
```

Comment:
Just wondering... isn't `complete_registration_attach_ical` a bit verbose? `attach_ical` would probably be enough...

Reply:
I added the location in case we want to attach ical files to any other registration-related emails, but might be unnecesary.